projects/1.20/assets/weapons-of-miracles-epicfight/wom/lang/zh_cn.json

```
@@ -0,0 +1,183 @@
+{
+	"itemGroup.wom.items": "奇迹武装",
+	
+	"enchantment.wom.invigoration" : "精力充沛",
+	"enchantment.wom.invigoration.desc" : "每级加2点最大耐力值",

```

```suggestion
	"enchantment.wom.invigoration.desc" : "每级增加2点最大耐力值。",
```
不清楚附魔描述模组一般是什么格式，这里暂以英文原文带句号的模式审阅

全文通查